ocaml/xapi-consts/api_errors.ml

```diff
@@ -1322,8 +1322,9 @@ let bundle_repo_not_enabled = add_error "BUNDLE_REPO_NOT_ENABLED"
 
 let can_not_sync_updates = add_error "CAN_NOT_SYNC_UPDATES"
 
-let bundle_repo_should_be_single_enabled =
-  add_error "BUNDLE_REPO_SHOULD_BE_SINGLE_ENABLED"
+let can_not_periodic_sync_updates = add_error "CAN_NOT_PERIODIC_SYNC_UPDATES"
+
+let repo_should_be_single_enabled = add_error "REPO_SHOULD_BE_SINGLE_ENABLED"
```

Comment:
The wording is a bit odd. Does this mean that only one repo should be enabled? "repo should be single enabled" is not grammatical and ambiguous. "repo should be the single one enabled" could be something I would guess.

Reply:
> Does this mean that only one repo should be enabled?
If the bundle repository or remote_pool repository is enabled, it should be the only one enabled repository of the pool. Or else, multiple repos (only remote) can be enabled.

Reply:
I agree renaming it to `repo_should_be_the_single_one_enabled` would be more clearer.